### DISCUSSION TITLE
problem with code server password

### DISCUSSION THREAD

**Discussion by Said113** [coder/code-server#3274] [Q&A] [answered]
i can't set a password to my code server

export PASSWORD=Ss399
code-server --auth password

in browser show wrong password

**Comment by code-asher** [comment, marked as answer]
Just to make sure that code-server is reading the environment properly could you try this and see if it works?

```
PASSWORD=Ss399 code-server --auth password
```

**Reply by Said113** [reply]
yes it's works now, thank you

**Reply by code-asher** [reply]
Awesome! You can also set the password in the config file `~/.config/code-server/config.yaml` if you'd rather do that instead of having it inline. 

If you want to use the `export` syntax you can post your setup and we can figure out how to make it work.
